cypress/integration/examples/dummy.spec.js

```diff
@@ -0,0 +1,22 @@
+/// <reference types="cypress" />
+
+context('Home page', () => {
+  beforeEach(() => {
+    cy.visit('localhost:3000')
+  })
+
+  it('should have a button to add salary in desktop version', () => {
+    cy.get('[data-cy=add-salary-button]').should('be.visible')
```

Comment:
We haven't talked about this. Are we going to remove the data attributes that are for `dev` purposes from the final build? I think we can do it using the `process.env`, but I'm not sure how to do it.

Reply:
Whatever you think is better, we can use regular selectors as well, but they are less maintainable
What's the problem of shipping the code with this data attributes (just curiosity)?

Reply:
I do not think is a bad idea to use `data-type` attributes, in fact it is what they recommend at Cypress. But I am open for discuss this.
https://docs.cypress.io/guides/references/best-practices.html

Reply:
I was doing some reading, and we can also use use `data-testid` then we will also have ours components ready for unit testing with `@testing-library/react` in case is needed.